release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -170,6 +170,19 @@ If a third-party CSI driver is present in the cluster, {product-title} does not
 Components with versions earlier than those above are still supported, but are deprecated. These versions are still fully supported, but version 4.12 of {product-title} requires vSphere virtual hardware version 15 or later. For more information, see xref:../release_notes/ocp-4-12-release-notes#ocp-4-12-deprecated-removed-features[Deprecated and removed features].
 
 Failing to meet the above requirements prevents {product-title} from upgrading to {product-title} 4.13 or later.
+[id="ocp-4-12-cluster-capabilities"]
```

Comment:
Approvalss for this change are in PR https://github.com/openshift/openshift-docs/pull/54585/